tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -405,9 +406,29 @@ def _submit_batch_job(
         else:
             return self._remote_command_executor.run_remote_command(submission_command, raise_on_error=raise_on_error)
 
+    def _dump_job_output(self, job_info):
+        match = re.match(r".*StdErr=(.*)?\sStdIn=(.*)?\sStdOut=(.*)?\sPower.*", job_info)
```

Comment:
There's a couple ways I'd simplify this.

1. This regex assumes that `StdErr` will always be before `StdIn`, which is always before `StdOut`, which is always before `Power`. Instead of relying on those assumptions I would loop through a list of only the strings we care about ('StdOut' and 'StdErr') and do the regex matching for each of them.
2. Use `re.search` instead of `re.match` so that you don't have to worry about not matching at the beginning of the string.

Reply:
Done